flit_core/bootstrap_install.py

```diff
@@ -37,12 +37,23 @@ def extract_wheel(whl_path, dest):
         default=purelib,
         help=f'installdir directory (defaults to {purelib})',
     )
+    parser.add_argument(
+        '--install-root',
+        type=Path,
+        default=None,
+        help='if given, installdir is considered to be under this'
+    )
 
     args = parser.parse_args()
 
     if not args.wheel.name.startswith('flit_core-'):
         sys.exit("Use this script only for flit_core wheels")
-    if not args.installdir.is_dir():
-        sys.exit(f"{args.installdir} is not a directory")
-
-    extract_wheel(args.wheel, args.installdir)
+    if args.install_root:
+        installdir = args.install_root / args.installdir.relative_to("/")
+        installdir.mkdir(parents=True, exist_ok=True)
```

Comment:
Let's do the mkdir in either case, for consistency. I don't think there was a particular reason why I was checking for an existing directory - maybe I just wanted to ensure it failed with a clear error if the expectations weren't met.

Reply:
We have to, the point with install-root is that install-root *does* exist but installdir *does not*. This is essentially a vanilla destination that is captured and packaged further.

Reply:
I see. Well, if I can remove the check, then that would maybe work too.